### DISCUSSION TITLE
Documentation

### DISCUSSION THREAD

**Discussion by tresorkalanda-chd** [laravel/framework#48907] [Show and tell]
![image](https://github.com/laravel/framework/assets/144696874/20fe6dbf-cfa6-439c-8489-6510d3fef2e1)
In this section impossible to read

**Comment by Sophist-UK** [comment]
A URL would be helpful.

**Comment by antimech** [comment]
I guess this is the page: https://laravel.com/docs/10.x/session#implementing-the-driver
![](https://github.com/laravel/framework/assets/13885653/50e679c1-03e6-4e77-8eff-db5c723086a6)
But it works for me. Try these:
1. Hard reload (clear cache + reload);
2. Browser incognito mode (without extensions);
3. Another browser.
